Cargo.toml

```diff
@@ -36,10 +36,10 @@ nix = "0.14"
 base64 = "0.10"
 task_scheduler = "0.2.0"
 structopt = "0.2.18"
-
 # Statically link SQLite (use the crate version provided by Diesel)
 # The highest version which Diesel currently allows is 0.12.0
 libsqlite3-sys = { version = "0.12.0", features = ["bundled"] }
+ldap3 = "0.6.1"
```

Comment:
This transitively adds a requirement on OpenSSL, which needs some native bindings. The docker build containers will need to be updated to support this requirement.